pandas/tests/frame/test_constructors.py

```diff
@@ -2657,6 +2657,12 @@ def test_construct_with_strings_and_none(self):
         expected = DataFrame({"a": ["1", "2", None]}, dtype="str")
         tm.assert_frame_equal(df, expected)
 
+    def test_construct_from_dict_ea_series(self):
```

Comment:
Small nit, but there is a `test_dict_nocopy` test a bit above that is specifically testing the behaviour of copy=True vs False for DataFrame(dict). Maybe can move the test to just below that one?

Reply:
Done